### PR TITLE
add README, prettier, config, license, etc.

### PR DESCRIPTION
**Pull request by notrab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<p align="center">
+  <a href="https://grafbase.com">
+    <img src="https://grafbase.com/images/other/grafbase-logo-circle.png" height="96">
+    <h3 align="center">Grafbase</h3>
```
Comment (yoav-lavi) [inline README.md:4]: Wouldn't we want a different header here and tagline below?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,5 @@
+root = true
```
Comment (fbjork) [inline .editorconfig:1]: do we want this file committed?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| [Next.js + Clerk](/nextjs-clerk)                  | Use [Clerk](https://clerk.dev/integrations/grafbase) as your Identity Provider with Grafbase.                                                                                            |
+| [React Apollo + Live Queries](/react-apollo-live) | Use [GraphQL Live Queries](https://grafbase.com/docs/realtime/live-queries) with [React](https://reactjs.org), [Apollo Client](https://www.apollographql.com/docs/react/), and Grafbase. |
+| [React URQL + Live Queries](/react-urql-live)     | Use [GraphQL Live Queries](https://grafbase.com/docs/realtime/live-queries) with [React](https://reactjs.org/), [URQL](https://formidable.com/open-source/urql/), and Grafbase.          |
+
+## Learn more
+
+- <a href="https://grafbase.com/community"><strong>Join the community</strong></a>
+- <a href="https://grafbase.com/docs"><strong>Documentation</strong></a>
+- <a href="https://grafbase.com/changelog"><strong>Changelog</strong></a>
+- <strong>[CLI](https://github.com/grafbase/cli)</strong>
```
Comment (yoav-lavi) [inline README.md:46]: AFAIK we don't plan to rename the CLI repo

File: .editorconfig (hunk 2 above)
Comment (notrab) [reply .editorconfig:1]: It does no harm here (and in the other one) for the code examples. It ensures a few sensible defaults.

Comment (yoav-lavi) [reply .editorconfig:1]: Yeah it's like a prettier config but more general

Comment (yoav-lavi) [reply .editorconfig:1]: We have this for the CLI repo too

File: README.md (hunk 3 above)
Comment (fbjork) [reply README.md:46]: We should link to the grafbase.com/cli page here

Comment (notrab) [reply README.md]: typo. Fixed. thanks!

File: README.md (hunk 1 above)
Comment (notrab) [reply README.md]: I pushed an update

**Review by yoav-lavi — APPROVED** [review]
Looks good!

**Comment by yoav-lavi** [comment]
See also https://linear.app/grafbase/issue/GB-2678/additional-open-source-repo-resources

**Pull request merged**